src/_actions/SettingsActions.js

```diff
@@ -19,3 +19,8 @@ export const serverDataAccountSettings = serverResponse => ({
     type: types.SERVER_DATA_ACCOUNT_SETTINGS,
     serverResponse,
 });
+
+export const updatePersonalAddress = newAddress => ({
```

Comment:
@borisyankov I only able to find `serverDataAccountSettings`, which I ignore it at first because semantically it perform something else (update store from server data) rather than (update store) which is more generic, so do you think I should create a generic action like what u mentioned?